rest-client/client-common/src/main/java/io/apicurio/registry/rest/client/impl/ErrorHandler.java

```diff
@@ -40,20 +39,20 @@
     private static final ObjectMapper mapper = new ObjectMapper()
             .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
 
-    private static final Logger logger = Logger.getLogger(BodyHandler.class.getName());
+    private static final Logger logger = Logger.getLogger(ErrorHandler.class.getName());
 
-    public static RestClientException handleErrorResponse(InputStream body, HttpResponse.ResponseInfo responseInfo) {
+    public static RestClientException handleErrorResponse(InputStream body, int statusCode) {
         try {
-            if (responseInfo.statusCode() == HttpStatus.SC_UNAUTHORIZED) {
+            if (statusCode == HttpStatus.SC_UNAUTHORIZED) {
```

Comment:
Looking at the imports, looks like `HttpStatus` is actually `org.apache.http.HttpStatus`? Wouldn't it be better to completely avoid depending on anything Apache HTTP to prevent it from accidentally creeping in?

Reply:
Agree, just inheritance from the old implementation, I will remove this.